cmd/nodecmd/loadtest.go

```diff
@@ -404,10 +411,32 @@ func createClusterYAMLFile(clusterName, subnetID, chainID string, separateHost *
 	return enc.Encode(clusterInfoYAML)
 }
 
+func getGitCommit(gitRepoURL string) string {
+	if strings.Contains(gitRepoURL, "/commit/") {
+		splitURL := strings.Split(gitRepoURL, "/")
+		commitID := splitURL[len(splitURL)-1]
+		return commitID
+	}
+	return ""
+}
+
+func getRepoFromCommitURL(gitRepoURL string) string {
+	splitURL := strings.Split(gitRepoURL, "/")
+	splitURLWOCommit := splitURL[:len(splitURL)-2]
+	repoDirName = splitURLWOCommit[len(splitURLWOCommit)-1]
+	gitRepo := strings.Join(splitURLWOCommit[:], "/")
+	gitRepo += ".git"
```

Comment:
nit: we have `const gitExtension = ".git"` maybe we should use it there too so we reuse const